src/XMonad/Core.hs

```diff
@@ -438,39 +440,124 @@ runOnWorkspaces job = do
              $ current ws : visible ws
     modify $ \s -> s { windowset = ws { current = c, visible = v, hidden = h } }
 
--- | Return the path to @~\/.xmonad@.
+-- | Return the path to the xmonad configuration directory.  Several
```

Comment:
Since there are now multiple xmonad directories (configuration, cache, data), it would be good to include here a quick description of what should go in the config directory, and under what circumstances it will be accessed.  I am not sure I completely understand this myself.